blueoil/datasets/fer_2013.py

```diff
@@ -74,46 +74,43 @@ def num_per_epoch(self):
 
         return len(self.images)
 
-    def _load_data(self, path):
+    def _load_data(self, path, subset):
```

Comment:
Sorry, I forgot to point this out. You don't need to pass `subset` as an argument. You can access it as `self.subset`.